docs/quickstart-install.md

```diff
@@ -6,7 +6,61 @@ Install the Nillion SDK, including the <strong>`nada`</strong> tool you'll use t
 
 ## Use nilup to install the Nillion SDK
 
-<SdkInstallation/>
+
+:::info
+Nillion SDK binaries are available for [Linux and macOS platforms](/limitations#platforms). For Windows installation, make sure to follow our [Windows developer environment setup](/installation#windows-guide) guide ahead of installing binaries.
+
+Prerequisties:
+- Python (>= 3.10) 
+- Node (>= v18.17)
+:::
+
+We have made this a simpler process with our NPX tool: `create-nillion-app`. 
+
+In one command it installs:
+- Nillion SDK
+- A NextJS app
+- Python / Nada folders to use Nada programs
+
+# Getting Started
+
+1. Run `npx @nillion/create-nillion-app` and then follow the CLI instructions
```

Comment:
nit: put the npx command in it's own line so Docusaurus gives it the copy/paste functionality like:

```bash
npx @nillion/create-nillion-app
```

Reply:
Great pick up!